docs/Best Practice Documents/1.1. Establish Session/index.md

```diff
@@ -0,0 +1,97 @@
+
+### 3.4. Changes in the launch sequence of MEDIA type App
+The following are additional behaviors required in the red frame of Figure1 in case of the MEDIA type app will be launched.
+When the SDL App receives `OnHMIStatus` with HMI Level FULL, the HMI displays the SDL App on foreground of the HU screen. After the app is displayed on the foreground, the SDL App sets the HMI template by sending `SetDisplaylayout`. Then, after the SDL App receives `showResponse`, it sets a progress bar on the HU screen by sending `SetMediaClockTimer`. The SDL App sends `SubscribeButton` and monitors the HMI Button event.
+
+### 3.5. Changes in the launch sequence of NAVIGATION/PROJECTION type App
+The following additional/excluded behaviores are required in the red frame of Figure1 in case the NAVIGATION/PROJECTION type app is launched.
+When the SDL App receives `OnHMIStatus` with HMI Level FULL, the HMI displays the SDL App on foreground of the HU screen. After the App is displayed on the foreground, the SDL App does NOT send `show`.
+
+<b>[The following applies only to the Android version of SDL App]</b>
+After the app is displayed to the foreground, (the SDL App) starts VideoStreamingManager/AudioStreamingManager. Then, when VideoStreamingManager/AudioStreamingManager has started successfully, it sends the video/audio data to the HU.
+
+
+## 4. Differences from the SDL standard specification
+The red frame in Figure1, which is supposed to be an essential part of the sequence, is not explicitly defined in the SDL standard specification. Therefore, all of the contents describe in 3.3 - 3.4 differ from the existing SDL standard specification.
+
+
+## 5. Sequence Diagrams
+The three types of launch sequence are described based on the SDL App launch sequence (Figure1) published in the SDL Standard Specification.
+&nbsp;(1) Normal type App launch sequence (Figure2)
+&nbsp;(2) MEDIA type App launch sequence (Figure3)
+&nbsp;(3) NAVIGATION / PROJECTION type App launch sequence (Figure4)
+
+Additional sequence is added in the red frame of Figure1 for each App type. Parts other than the red frame are common to/shared by all App type, and there are no lack in the sequences. The sequence described in the red part refers to the RPC communication sequence; it is not described in the SDL standard specification.
+
+|||
+**Figure2.** Normal type App launch sequence
+![Figure2_Normal_type_App_launch_sequence.png](./assets/Figure2_Normal_type_App_launch_sequence.png)
+|||
+
+|||
+**Figure3.** MEDIA type App launch sequence
+![Figure3_MEDIA_type_App_launch_sequence.png](./assets/Figure3_MEDIA_type_App_launch_sequence.png)
+|||
+
+|||
+**Figure4.** NAVIGATION / PROJECTION type App launch sequence
+![Figure4_NAVIGATION-PROJECTION_type_App_launch_sequence.png](./assets/Figure4_NAVIGATION-PROJECTION_type_App_launch_sequence.png)
```

Comment:
@JackLivio -san, Initialization processing of StreamingManager of iOS application is to be executed inside proxy, but it is not processed by Android proxy.  Therefore, after registering app, it will be as follows.
For iOS apps: The app developer doesn't need to do anything.
For Android apps: The app developer must perform StreamingManager initialization by himself to create a streaming session.